### PR TITLE
Move to Polymer 3.0, and start move to LitElement + MWC

### PR DESCRIPTION
**Pull request by kenchris**

### CODE REVIEW

**Comment by kenchris** [comment]
Some files are missing

image

**Comment by rakuco** [comment]
> Some files are missing

They seem to have been overlooked in #35 and #39. @honry, do you still have those around by any chance?

**Comment by Honry** [comment]
Issue link: https://github.com/intel/generic-sensor-demos/issues/62

**Comment by rakuco** [comment]
While you're still at it: I guess you also need to drop all references to Bower from `README.md`, right?

**Comment by rakuco** [comment]
@kenchris has told me this is finally ready for merging, so let's go!

File: sensor-tester/src/sensor-tester.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+import '@polymer/app-layout/app-drawer/app-drawer';
+import '@polymer/app-layout/app-drawer-layout/app-drawer-layout';
+import '@polymer/app-layout/app-header/app-header';
+import '@polymer/app-layout/app-header-layout/app-header-layout';
+import '@polymer/app-layout/app-scroll-effects/app-scroll-effects';
+import '@polymer/app-layout/app-toolbar/app-toolbar';
+import {LitElement, html} from '@polymer/lit-element';
+import {installRouter} from '../node_modules/pwa-helpers/router.js';
+
+const menuIcon = html`<svg height="24" viewBox="0 0 24 24" width="24"><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path></svg>`;
```
Comment (alexshalamov) [inline sensor-tester/src/sensor-tester.js]: Should be same color as the app header text color.

**Pull request merged**